src/App/Part.cpp

```diff
@@ -104,6 +105,37 @@ PyObject *Part::getPyObject()
     return Py::new_reference_to(PythonObject);
 }
 
+std::vector<DocumentObject*> Part::addObjects(std::vector<DocumentObject*> objs)
+{
+    // note that addObjects inherits from OriginGroupExtension while
+    // removeObjects from GeoFeatureGroupExtension
+    std::vector<DocumentObject*> addedObjs = OriginGroupExtension::addObjects(objs);
+    
+    for(auto obj : addedObjs) {
+        // if of type VarSet, enable exposed
+        if (obj->isDerivedFrom<App::VarSet>()) {
```

Comment:
Since dynamic_cast is used below this line is superfluous and it can be directly written
`if (auto varSet = dynamic_cast<App::VarSet*>(obj))`

Reply:
Thank you, yes of course, you're absolutely right.